charts/replicated-library/values.yaml

```diff
@@ -259,14 +259,16 @@ services:
   example:
     # -- Enables or disables the service
     enabled: false
-    # -- Name of the app to attach this service. This corresponds to an app configured un the `apps` key
-    # TODO: Accept a list of appnames of which to associate the service
-    # TODO: Needs to be optional
-    appName: example
+    # -- Optional list of apps to attach this service. This corresponds to apps configured in the `apps` key
+    appName:
+    - example
     # -- Override the name of this object. Default name if not overwritten will be releaseName-ChartName-objectName
     fullNameOverride:
     # -- Set the service type
     type: ClusterIP
+    # -- Set the list of selectors for this object. this takes precedence over appName.
+    selector:
+      app: my-app
```

Comment:
I think it might be better to just do the same thing that was done with annotations and labels below and just clarify that it accepts an object. The reason being, when I see this in the readme above, "app" and "my-app" showing in object format left me thinking it meant something else. Seeing it here it's more clear this is a selector named app with the value my-app but it really threw me in the readme.